### DISCUSSION TITLE
2007/08/01/%E4%B8%80%E5%B9%B4%E7%9A%84-labview-%E5%86%99%E4%BD%9C%E6%88%90%E7%BB%A9

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#174] [Announcements]
# 2007/08/01/%E4%B8%80%E5%B9%B4%E7%9A%84-labview-%E5%86%99%E4%BD%9C%E6%88%90%E7%BB%A9

我开始决定整理和写作 LabVIEW 的相关知识，恰好是一年之前。今天回头查看了一下自己到底写了多少东西。

https://blog.qizhen.xyz/2007/08/01/%E4%B8%80%E5%B9%B4%E7%9A%84-labview-%E5%86%99%E4%BD%9C%E6%88%90%E7%BB%A9

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2008年05月29日 09:28 编辑
呵呵，出书需要花费很大的精力。我是比较懒惰的，现在还不想在这上面花太多时间:)

回复
野鸭
2008年05月28日 22:49 编辑
支持出书！

回复
国强
2007年08月2日 15:32 编辑
刚开始学啊，以后阮兄多指教

回复
arrowpig
2007年08月1日 12:58 编辑
这么快，已经一年了！！！你这一年很成功，老婆也娶好了。而且再也不是“蠢驴”了~~ 

回复
永青
2007年08月1日 10:33 编辑
大家就等你出书了，老大 

回复
li
2007年08月1日 10:04 编辑
　　　不错，不错
继续加油．
